controllers/ramenconfig.go

```diff
@@ -231,6 +188,26 @@ func ConfigMapGet(
 	return
 }
 
+func RamenConfigUpdate(
+	log logr.Logger,
+	configmap client.Object,
+) (ramenConfig *ramendrv1alpha1.RamenConfig, err error) {
+	log.Info(fmt.Sprintf("Update in %s configuration map", configmap.GetName()))
+
+	configMap, ok := configmap.(*corev1.ConfigMap)
+	if !ok {
+		err = fmt.Errorf("config map function received non-configMap resource %s",
+			configMap.GetName())
```

Comment:
We potentially need to use `configmap` here and not `configMap` as the casting failed.

Maybe better to rename `configmap` as `inObj` or so for distinction.